zebra-state/src/service/tests.rs

```diff
@@ -0,0 +1,182 @@
+use std::sync::Arc;
+
+use futures::stream::FuturesUnordered;
+use tower::{util::BoxService, Service, ServiceExt};
+use zebra_chain::{
+    block::Block, parameters::Network, serialization::ZcashDeserializeInto, transaction,
+    transparent,
+};
+use zebra_test::{prelude::*, transcript::Transcript};
+
+use crate::{init, BoxError, Config, Request, Response};
+
+const LAST_BLOCK_HEIGHT: u32 = 10;
```

Comment:
We could add a function to `zebra_test` that returns the continuous range of blocks from genesis.

But we don't have to do that in this PR, let's open a follow-up ticket that isn't in the first alpha?